.buildkite/gpu_pipeline/pipeline.yml

```diff
@@ -50,3 +50,14 @@ steps:
         agents:
           slurm_gpus: 1
           slurm_time: 23:00:00
+
+      - label: "target_gpu_implicit_baroclinic_wave_4process"
+        command: >
+          julia --project -e 'using CUDA; CUDA.versioninfo()'
+          
+          srun julia --color=yes --project=examples examples/hybrid/driver.jl
```

Comment:
Can we keep the `nsys profile`? You can use multi-line yaml to make it easier to read: https://github.com/CliMA/slurm-buildkite/wiki/Tips#splitting-commands-over-multiple-lines